flexmeasures/ui/templates/assets/asset_properties.html

```diff
@@ -245,152 +258,1077 @@ <h3>All child assets for {{ asset.name }}
+                        card.querySelector('.card-highlight').appendChild(carouselCards);
+                    };
+                    titleSubContainer.appendChild(plusIcon)
+                }
+
+
+                const iconSubContainer = document.createElement('div');
+                iconSubContainer.className = 'd-flex column-gap-2 justify-content-between align-items-center';
+
+                let fieldIconClass = '';
+                let fieldIconTitle = '';
+                if (isKeyValueObject) {
+                    fieldIconClass = iconMap["sensor"];
+                    fieldIconTitle = "A Sensor";
+                } else if (isBool) {
+                    fieldIconClass = iconMap["boolean"];
+                    fieldIconTitle = "A Boolean";
+                } else if (isNumber || isString) {
+                    fieldIconClass = iconMap["string"];
+                    fieldIconTitle = "A Fixed Value";
+                } else if (isArray) {
+                    fieldIconClass = iconMap["array"];
+                    fieldIconTitle = "An Array";
+                } else {
+                    fieldIconClass = iconMap["default"];
+                    fieldIconTitle = "Unknown";
+                }
+
+                const identityIcon = document.createElement('i');
+                identityIcon.className = `${fieldIconClass} ps-1 fa-md`;
+                identityIcon.setAttribute('data-bs-toggle', 'tooltip');
+                identityIcon.setAttribute('data-bs-html', 'true');
+                identityIcon.setAttribute('title', fieldIconTitle);
+                identityIcon.style.cursor = 'pointer';
+
+                const closeIcon = document.createElement('i');
+                closeIcon.className = 'fa fa-times card-close-icon';
+                closeIcon.setAttribute('data-bs-toggle', 'tooltip');
+                closeIcon.setAttribute('title', 'Unset this field');
+                closeIcon.style.cursor = 'pointer';
+                closeIcon.onclick = function () {
+                    // Remove this field
+                    if (activeCard() && activeCard().id === card.id) {
+                        setActiveCard(null); // Deselect if the same card is clicked again
+                        setSelectedIndex(null); // Reset selected index
+                        card.classList.remove('border-on-click'); // Remove the border class
+                    }
+
+                    card.remove(); // Remove the card from the DOM
+                    const flexModel = getFlexModel();
+                    if (Array.isArray(flexModel[name])) {
+                        flexModel[name] = [];
+                    } else {
+                        flexModel[name] = null;
+                    }
+
+                    // Update the flex model
+                    setFlexModel(flexModel);
+                    updateFlexModel();
+                }
+
+
+                iconSubContainer.appendChild(identityIcon);
```

Comment:
I like the `identityIcon` - the only thing I'd change is that it is too close to a main action (`closeIcon`), obscuring it a bit.

Can it also appear in the bottom right (not top right), and also be grayed out a little?